app/controllers/contributors/new.js

```diff
@@ -8,9 +8,7 @@ export default Controller.extend({
   actions: {
 
     save (contributor) {
-      return this.kredits
-                 .addContributor(contributor)
-                 .then(() => this.transitionToRoute('index'))
```

Comment:
Does this stay on the form page now, or is the transition happening somewhere else when editing or creating a new contibutor?

Reply:
Yes, I moved it to the component:

https://github.com/67P/kredits-web/pull/156/files#diff-05b2a32d1311a3737ec444f68955ccedR81